components/_host-profiles.scss

```diff
@@ -15,7 +24,9 @@ host profiles.
 }
 
 .photo-container {
-
+	position: relative;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered margin-bottom, position